packages/common/src/visualizations/helpers/styles/themeColors.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Theme color constants for ECharts styling
+ * Centralizes color usage across all chart style utilities
+ * Using Mantine default color palette
+ */
+
+// Gray scale
+export const GRAY_0 = '#f8f9fa'; // theme.colors.gray[0]
+export const GRAY_1 = '#f1f3f5'; // theme.colors.gray[1]
+export const GRAY_2 = '#e9ecef'; // theme.colors.gray[2]
+export const GRAY_3 = '#dee2e6'; // theme.colors.gray[3]
+export const GRAY_4 = '#ced4da'; // theme.colors.gray[4]
+export const GRAY_5 = '#adb5bd'; // theme.colors.gray[5]
+export const GRAY_6 = '#868e96'; // theme.colors.gray[6]
+export const GRAY_7 = '#495057'; // theme.colors.gray[7]
+export const GRAY_8 = '#343a40'; // theme.colors.gray[8]
+export const GRAY_9 = '#212529'; // theme.colors.gray[9]
```

Comment:
not the best way of doing it, but couldn't think of a better/cleaner approach. 

issue before: 
we were passing `theme` to these utils on the frontend, to get the gray colors.
Then, since I wanted to share the style utils between explorer(all code mostly on `frontend` package) and sql runner (all echarts option code in `common` package), I had to
- copy the mantine theme colors into this common file
- removed the `theme` dependency from the utils.